memory/memhotplug.py

```diff
@@ -53,14 +53,16 @@ def offline(block):
         pass
 
 
-def get_hotpluggable_blocks(path):
+def get_hotpluggable_blocks(path, ratio):
     mem_blocks = []
     for mem_blk in glob.glob(path):
         block = re.findall(r"\d+", os.path.basename(mem_blk))[0]
         block = re.sub(r'^\s*$', '', block)
         if memory.is_hot_pluggable(block):
             mem_blocks.append(block)
-    return mem_blocks
+    f = lambda num: (num / 100 + 1) if num % 2 else (num / 100)
+    count = f(len(mem_blocks) * ratio)
```

Comment:
lambda is prettier than the function like below:

`   def f(num):
        if num % 2:
            return (num / 100 + 1)               
        return  (num / 100)
count = f(len(mem_blocks) * ratio)
`

Reply:
@abdhaleegit The following might be the only reason for its inclusion from https://www.python.org/dev/peps/pep-0008/#programming-recommendations. 
```
Always use a def statement instead of an assignment statement that binds a lambda expression directly to an identifier.

Yes:

def f(x): return 2*x
No:

f = lambda x: 2*x
The first form means that the name of the resulting function object is specifically 'f' instead of the generic '<lambda>'. This is more useful for tracebacks and string representations in general. The use of the assignment statement eliminates the sole benefit a lambda expression can offer over an explicit def statement (i.e. that it can be embedded inside a larger expression)
```